.gitignore

```diff
@@ -1,3 +1,5 @@
+# Docker ignore files
+.dockerignore
```

Comment:
I don't think the .dockerignore should be in .gitignore. The .dockerignore should be the same across computers, and when building on Github Actions

Reply:
Agreed